api/src/main/java/org/pmiops/workbench/conceptset/ConceptSetMapper.java

```diff
@@ -0,0 +1,20 @@
+package org.pmiops.workbench.conceptset;
+
+import org.mapstruct.Mapper;
+import org.mapstruct.Mapping;
+import org.pmiops.workbench.db.model.DbConceptSet;
+import org.pmiops.workbench.model.ConceptSet;
+import org.pmiops.workbench.utils.mappers.CommonMappers;
+
+@Mapper(
+    componentModel = "spring",
+    uses = {CommonMappers.class})
+public interface ConceptSetMapper {
+
+  @Mapping(target = "id", source = "conceptSetId")
+  @Mapping(target = "domain", source = "domainEnum")
+  @Mapping(target = "survey", source = "surveysEnum")
+  @Mapping(target = "etag", source = "version", qualifiedByName = "etag")
+  @Mapping(target = "concepts", ignore = true)
+  ConceptSet dbModelToClient(DbConceptSet source);
+}
```

Comment:
Since you're. ignoring the concepts field, you can make a helper method that knows how to do what  you need. See e.g. `org.pmiops.workbench.utils.WorkspaceMapper#afterWorkspaceIntoResearchPurpose()`

Reply:
this can be done with this ticket: https://precisionmedicineinitiative.atlassian.net/browse/RW-4451